src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/CSharpTestLspServerHelpers.cs

```diff
@@ -28,7 +28,7 @@ internal class CSharpTestLspServerHelpers
 
             var exportProvider = RoslynTestCompositions.Roslyn.ExportProviderFactory.CreateExportProvider();
             var workspace = CreateCSharpTestWorkspace(files, exportProvider);
-            var clientCapabilities = new ClientCapabilities();
+            var clientCapabilities = new VSInternalClientCapabilities { SupportsVisualStudioExtensions = true };
```

Comment:
Roslyn returns a `VSInternalHover` type only if the `SupportsVisualStudioExtensions` property is set.